code/modules/client/preferences.dm

```diff
@@ -1509,16 +1509,20 @@ GLOBAL_LIST_EMPTY(preferences_datums)
 
 	character.backbag = backbag
 
-	character.dna.features = features.Copy()
-	character.dna.real_name = character.real_name
 	var/datum/species/chosen_species
 	if(!roundstart_checks || (pref_species.id in GLOB.roundstart_races))
 		chosen_species = pref_species.type
 	else
 		chosen_species = /datum/species/human
 		pref_species = new /datum/species/human
 		save_character()
+
 	character.set_species(chosen_species, icon_update = FALSE, pref_load = TRUE)
+	character.dna.features = features.Copy()
+	character.dna.real_name = character.real_name
+	
+	if(!features["tail_lizard"] == "None")

```

Comment:
`!something` will only ever be `0` or `1`, never `"None"`

Reply:
Yeah you need another layer of parentheses here.

Reply:
Well that's embarrassing, sorry about that should all be fix up in a moment